bundles/SystemInfoBundle/src/Controller/SettingsController.php

```diff
@@ -61,7 +61,7 @@ public function checkAction(Request $request, Connection $db, ?Profiler $profile
         }
 
         $viewParams = Requirements::checkAll($db);
-        $viewParams['headless'] = $request->query->getBoolean('headless') || $request->request->getBoolean('headless');
+        $viewParams['headless'] = (bool)$request->get('headless');
```

Comment:
Why did you revert this? The request->get() method is internal and shouldn't be used

Reply:
```suggestion
        $viewParams['headless'] = $request->query->getBoolean('headless') || $request->request->getBoolean('headless');
```

Reply:
Ahh, it was a merge conflict. thanks for the catch!